docs/client.rst

```diff
@@ -894,6 +894,27 @@ The request will be sent in the next ``run_session`` function call.
 
 ------
 
+.. code-block:: c
+    bool uxr_prepare_output_stream_fragmented(uxrSession* session, uxrStreamId stream_id, uxrObjectId datawriter_id,
+                                  struct ucdrBuffer* ub, size_t topic_size, uxrOnBuffersFull flush_callback);
```

Comment:
struct *ucdr buffer ub? ub_topic? mb? mb_topic?

Reply:
the name does not mind... just the type. In any case, in the actual library they have different names.

Reply:
Yes, of course. I mean, just to be consistent w/ the description given below.

Reply:
Done